### ISSUE TITLE
Investigation should not have inclusion_criteria and exclusion_criteria

### ISSUE THREAD

**Issue by bcorrie** [airr-knowledge/issues#62]
In the Consolidated spreadsheet (and in the schema) `Investigation` has slots `inclusion_criteria` and `exclusion_criteria`.

https://github.com/airr-knowledge/ak-schema/blob/506539c16f30a5e0ae1a2d58243c9288c27144a6/src/ak_schema/schema/ak_study_design.yaml#L15

At the same time `StudyArm` has slots `inclusion_criteria` and `exclusion_criteria` and refers to an `Investigation`.

https://github.com/airr-knowledge/ak-schema/blob/506539c16f30a5e0ae1a2d58243c9288c27144a6/src/ak_schema/schema/ak_study_design.yaml#L49

I believe this is an error as these are redundant. I think it makes the most sense to have these in `StudyArm`

I have highlighted the fields in Red in the consolidated spreadsheet.

**Comment by jamesaoverton** [comment]
I agree that `inclusion_criteria` and `exclusion_criteria` are most relevant to the `StudyArm`.

I can imagine `inclusion_criteria` and `exclusion_criteria` for an `Investigation` being used to summarize the criteria across study arms, for the study as a whole, but I would be fine without them.

**Comment by schristley** [comment]
@jamesaoverton With the data you are seeing in ImmPort, are `inclusion_criteria` and `exclusion_criteria` provided separately? In the AIRR standards, these are munged together into a single field.

**Comment by jamesaoverton** [comment]
For the study arms, ImmPort specifies inclusion and exclusion criteria, and calls them that. For studies/investigations, there are various fields that could be interpreted as inclusion/exclusion criteria, but are not phrased as such.

For example, compare the "Study" and "Design" tabs on this page: <https://www.immport.org/shared/study/SDY460>.

The upshot is that I don't need `inclusion_criteria` and `exclusion_criteria` slots on `Investigation` for my stuff, and I would have to combine multiple ImmPort fields to generate values for these, or manually curate them. But it will be easy to map ImmPort data to `inclusion_criteria` and `exclusion_criteria` slots on `StudyArm`.

**Comment by schristley** [comment]
> For example, compare the "Study" and "Design" tabs on this page: https://www.immport.org/shared/study/SDY460.

That's actually quite nice with it separated into a set of individual inclusion and exclusion criteria.

**Comment by LonnekeScheffer** [comment]
bringing this up now again since it came up in immuneSpace integration

> For the study arms, ImmPort specifies inclusion and exclusion criteria, and calls them that. For studies/investigations, there are various fields that could be interpreted as inclusion/exclusion criteria, but are not phrased as such.
> 
> For example, compare the "Study" and "Design" tabs on this page: https://www.immport.org/shared/study/SDY460.
> 
> The upshot is that I don't need `inclusion_criteria` and `exclusion_criteria` slots on `Investigation` for my stuff, and I would have to combine multiple ImmPort fields to generate values for these, or manually curate them. But it will be easy to map ImmPort data to `inclusion_criteria` and `exclusion_criteria` slots on `StudyArm`.

@jamesaoverton am I misunderstanding this? In ImmuneSpace, I didn't find direct inclusion/exclusion criteria for the arms (only 'description' which can be viewed as inclusion_criteria). However, in immPort, there is the inclusion_exclusion table listing criteria (the same as you linked above, under 'design') which I interpreted as applying to the full study/investigation level? 

For instance: one of the listed inclusion criteria is "Age 18-30, 60-79, or 80-100 years, inclusive at time of initial enrollment". That applies to the investigation, and includes all 3 arms (one arm for 18-30, one for 60-79 and the last arm 80-100)

**Comment by bcorrie** [comment]
> For instance: one of the listed inclusion criteria is "Age 18-30, 60-79, or 80-100 years, inclusive at time of initial enrollment". That applies to the investigation, and includes all 3 arms (one arm for 18-30, one for 60-79 and the last arm 80-100)

The ADC to AKC conversion will have the same problem. In the ADC there is a single field called `inclusion_exclusion_crtieria` which is a string which has both, written in "ad-hoc" sentences. 

I feel that the AKC has the best definition if I recall correctly. Each Study Arm has an `inclusion_criteria` and `exclusion_criteria`. Ideally this would be written in some formal language like `{"includes":["age_min":18,"age_max":30]}` but that is perhaps over the top for our purposes.

This ADC data is likely to be impossible to parse (at least 100%) when converting to the AKC, not sure what we will do. Currently I think I copy the `inclusion_exclusion_crtieria` into both AKC `inclusion_criteria` and `exclusion_criteria` fields.

**Comment by jamesaoverton** [comment]
I think it's more useful to attach inclusion and exclusion criteria to arms. For ImmuneSpace we've been curating an 'arm.arm_description' column that's pretty close but not quite the same. I'll ask about this.

**Comment by LonnekeScheffer** [comment]
My take-away from today's meeting:

- We keep inclusion_criteria & exclusion_criteria on the study/investigation level. This information is moreso retained for completeness (since both immPort and MiAIRR studies provide it) but it may not be easily queryable. 
- On the arm level, we would ideally have something queryable (e.g. like ['filters' defined for RepertoireGroups in AIRR](https://github.com/airr-community/airr-standards/issues/578#issuecomment-1975265885)). But this _requires manual curation_ as such info cannot be automatically extracted. 

I would suggest not to use the names inclusion_criteria and exclusion_criteria on the study arm level, to avoid confusion? Maybe for now we can keep a study arm 'arm_description' (string) and refine this later to add something like filter terms?

**Comment by schristley** [comment]
> My take-away from today's meeting:
> 
> * We keep inclusion_criteria & exclusion_criteria on the study/investigation level. This information is moreso retained for completeness (since both immPort and MiAIRR studies provide it) but it may not be easily queryable.
> * On the arm level, we would ideally have something queryable (e.g. like ['filters' defined for RepertoireGroups in AIRR](https://github.com/airr-community/airr-standards/issues/578#issuecomment-1975265885)). But this _requires manual curation_ as such info cannot be automatically extracted.
> 
> I would suggest not to use the names inclusion_criteria and exclusion_criteria on the study arm level, to avoid confusion? Maybe for now we can keep a study arm 'arm_description' (string) and refine this later to add something like filter terms?

What I got from Bjoern's last comment was that there is a difference between the "planned" study arms and then what actually happens, so I think the `StudyArm` is the plan.  Recording the arm_description for what was planned sounds reasonable.

@LonnekeScheffer Don't we know which participants are in which study arm? Even though we might not have a computable filter, we can still assign participants to each? While the filter is nice, it would also be sufficient to "assign" each participant to an arm.

**Comment by bcorrie** [comment]
> What I got from Bjoern's last comment was that there is a difference between the "planned" study arms and then what actually happens, so I think the `StudyArm` is the plan. Recording the arm_description for what was planned sounds reasonable.

Don't we want to capture what actually happened in `StudyArm` rather than what the plan was. If the plan and the actual experiment differ I would think we would want the data in `StudyArm` to record what actually happened in the experiment, and therefore what the data actual represents, not what the plan was, no?

**Comment by schristley** [comment]
> > What I got from Bjoern's last comment was that there is a difference between the "planned" study arms and then what actually happens, so I think the `StudyArm` is the plan. Recording the arm_description for what was planned sounds reasonable.
> 
> Don't we want to capture what actually happened in `StudyArm` rather than what the plan was. If the plan and the actual experiment differ I would think we would want the data in `StudyArm` to record what actually happened in the experiment, and therefore what the data actual represents, not what the plan was, no?

Yes. I guess my point was that Bjoern mentioned both, the plan and the actual. We should know the actual participants in each study arm, so yes, we should assign them to the appropriate study arm. The inclusion/exclusion criteria is, I'm thinking, the planned criteria, but it sounds like ImmuneSpace doesn't really have that info. there just seems to be a description of the arm, so instead of trying to turn that description into actual inclusion/exclusion criteria, let's put those criteria slots aside and put the info in the description slot.

And looking at the `StudyArm` and `Participant` relationship, it only allows a participant to belong to one study arm. From an analysis perspective, this might be limiting and/or incomplete. Think about multiple specimens collected from the same participant whereby you want to analyze those specimens in different ways, so when it comes to actual comparison groups (e.g. RepertoireGroup) for analysis, the `StudyArm` is a bit too coarse.

**Comment by jamesaoverton** [comment]
Yes, the `StudyArm` is part of the study design, and should be thought of at that level.

I would want to hear a very strong case before allowing a `Participant` to belong to more than one `StudyArm`.

We have `Dataset` for collecting arbitrary groups for analysis (which may or may not correspond to an arm): https://github.com/airr-knowledge/ak-schema/blob/main/src/ak_schema/schema/ak_commons.yaml#L60

We have the specific `LifeEvent`s to describe what actually happened to a participant, regardless of plans.

**Comment by schristley** [comment]
> We have `Dataset` for collecting arbitrary groups for analysis (which may or may not correspond to an arm): 

Ok, that's a good start. We will need to think about how to properly extend `Dataset` to handle the variety of data and where the data is stored. Also if `Dataset` is intended to define groups for analysis, how is that specified.

I'm not sure if a `Dataset` object is supposed to represent not just the raw data coming from an assay but also encompasses any downstream processing of that data, or if we would have a new `Dataset` object for the output of each data processing step. I would assume the latter otherwise the `Dataset` becomes complex to represent all of that.

Lonneke and I were discussing extracting archival IDs (GEO, BioProject, SRA, etc.) and where that information goes. For example, my first thought is that an SRA ID which references specific raw sequencing file(s) from an assay would be put in `Dataset`.

Ffter posting this I realize it's digressing from the main issue, but it's important so I've added #78

**Comment by bcorrie** [comment]
> I would want to hear a very strong case before allowing a `Participant` to belong to more than one `StudyArm`.

Although far from an expert on study design, I think a simple case would be an investigation that was studying disease (Case) vs healthy (Control) tissue, and both disease and healthy tissue samples were taken from the same subject. 

Maybe even more basic would be an investigation that was looking at the differences between tissue types where the study arms were the different tissue types, and you want tissue from the same individual to control for individual differences.

Am I crazy, these both seem like logical study designs.

**Comment by LonnekeScheffer** [comment]
Brian, if you search online for how the term "study arm" is used; it always refers to a (sub)group of participants, which may receive specific interventions or treatments ([example1](https://mrctcenter.org/glossaryterm/arm/#:~:text=A%20group%20of%20participants%20in,receive%20the%20same%20study%20treatment.), [example2](https://toolkit.ncats.nih.gov/glossary/arm/)). This seems to be pretty standard terminology. What you're describing is fine as a study design but it just wouldn't be called study arm.

**Comment by LonnekeScheffer** [comment]
Regarding Bjoerns comments of "planned vs reality": the example he brought up was not necessarily that "things weren't done according to plan", but rather that even following the plan there may be some unintended bias. He brought up the age example: the plan could be Study Arm A = 20 - 30 years old, but in practice all participants in this arm are 20 - 22 years old. I don't think you would in this case replace the description of arm A from "20 - 30 years old" to "20 - 22", since the original label isn't wrong. Plus this information is still available and queryable, but on the participant level, not the arm level. 

If a participant actually ends up doing something that makes them not belong to the arm anymore (e.g., arm A should take medication X, and the person stops taking the medication due to side effects), I think that's a problem for the people governing that study themselves (e.g., excluding the participant?), not something we should have to resolve in the AKC on their behalf.

**Comment by bcorrie** [comment]
> Brian, if you search online for how the term "study arm" is used; it always refers to a (sub)group of participants, which may receive specific interventions or treatments ([example1](https://mrctcenter.org/glossaryterm/arm/#:~:text=A%20group%20of%20participants%20in,receive%20the%20same%20study%20treatment.), [example2](https://toolkit.ncats.nih.gov/glossary/arm/)). This seems to be pretty standard terminology. What you're describing is fine as a study design but it just wouldn't be called study arm.

Fair enough, but then I would argue our data model for describing an `Investigation` doesn't have the ability to describe important parts of the study design of many `Investigations` that we might see in the AKC. 8-) 

Admittedly, the ADC does not have the ability to do this either in any concise and structured way, but I would suggest it is a gap in our data model that we might want to consider???

**Comment by schristley** [comment]
> > I would want to hear a very strong case before allowing a `Participant` to belong to more than one `StudyArm`.
> 
> Although far from an expert on study design, I think a simple case would be an investigation that was studying disease (Case) vs healthy (Control) tissue, and both disease and healthy tissue samples were taken from the same subject.
> 
> Maybe even more basic would be an investigation that was looking at the differences between tissue types where the study arms were the different tissue types, and you want tissue from the same individual to control for individual differences.
> 
> Am I crazy, these both seem like logical study designs.

I believe you are mixing up the comparative analyses on samples that a study performs vs. the basic categorization that `StudyArm` provides for participants. There will be a different mechanism that defines what samples to compare, James suggested `Dataset` is more appropriate for this. `Dataset` needs more expansion (#78) to complete support that though.

> Although far from an expert on study design, I think a simple case would be an investigation that was studying disease (Case) vs healthy (Control) tissue, and both disease and healthy tissue samples were taken from the same subject.

This would be two study arms, one Case and one Control, but another mechanism would say "take the disease samples from Case and Control and compare them", or "take the disease and healthy samples from both Case and Control and compare them" and the various other analysis combinations.

> Maybe even more basic would be an investigation that was looking at the differences between tissue types where the study arms were the different tissue types, and you want tissue from the same individual to control for individual differences.

This might be just a single study arm, and again another mechanism would indicate what samples should be compared for analysis.

**Comment by bcorrie** [comment]
In meeting, discussed this and agreed that `StudyArm` is a concept used widely to track `Participants` role in an `Investigation` (e.g. Clinical Studies) and we should not mess with this concept. @jamesaoverton showed what looked like a nice model for modeling comparisons within an investigation from ImmuneSpace. @LonnekeScheffer what were the ImmuneSpace objects that captured comparisons within a study that James showed in the meeting? I think you mentioned that they should be in the AKC schema, but I am not sure I see them in the schema with this pull request (at least in the ak_schema directory).

Can you point these out to me...

**Comment by jamesaoverton** [comment]
@bcorrie The LinkML classes for `Dataset` and `Conclusion` are here: https://github.com/airr-knowledge/ak-schema/blob/main/src/ak_schema/schema/ak_specimens.yaml#L88

**Comment by bcorrie** [comment]
Ah, thanks, I was looking in ak_study_desing.yaml
